src/solvers/strings/string_constraint_generator.h

```diff
@@ -111,6 +111,11 @@ std::pair<exprt, string_constraintst> add_axioms_for_string_of_int_with_radix(
   const namespacet &ns,
   array_poolt &array_pool);
 
+/// Given an exprt and a string representing a character set of the form
+/// [low_char]-[high_char], return an and_exprt that represents restricting
+/// the selection of the character to the range in the character set.
+and_exprt char_set_restrictions(const exprt &expr, const std::string &char_set);
```

Comment:
:bulb: @svorenova added a range class for expressing the `low_char-high_car` that might be good to reuse